style.css

```diff
@@ -1 +1,345 @@
 /*  Add your styling here */
+
+body {
+  background-color: #f0d7d2;
+  color: #4285f4;
+  font-family: "Karla", sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Try and limit rules on the `body` to very generic and common styles, such as the background-color and font-family, but not flexbox rules (as it's unlikely your entire page should follow the same flex layout rules always)